modules/blocks/chain.js

```diff
@@ -251,7 +251,7 @@ Chain.prototype.applyGenesisBlock = function(block, cb) {
 		err => {
 			if (err) {
 				// If genesis block is invalid, kill the node...
-				process.exit(0);
+				process.emit('cleanup', err.message);
```

Comment:
Why is this change here? How is it related to the PR?

Reply:
During some testing, identified this use case where lisk exit without showing error. So fix it right there instead of creating a 1 line PR.